pkg/sqlmanager/schema_test.go

```diff
@@ -1,50 +1,65 @@
 package sqlmanager
 
 import (
-	embeddedpostgres "github.com/fergusstrange/embedded-postgres"
+	"fmt"
 	"github.com/nrc-no/core/pkg/api/types"
+	"github.com/nrc-no/core/pkg/mocks"
 	"github.com/nrc-no/core/pkg/sql/schema"
+	"github.com/nrc-no/core/pkg/testutils"
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/suite"
 	"gorm.io/driver/postgres"
 	"gorm.io/gorm"
+	"os"
 	"strings"
 	"testing"
 )
 
 type Suite struct {
 	suite.Suite
-	pg *embeddedpostgres.EmbeddedPostgres
-	db *gorm.DB
+	db   *gorm.DB
+	done func()
 }
 
 func TestSuite(t *testing.T) {
 	suite.Run(t, new(Suite))
 }
 
+func TestMain(m *testing.M) {
+	// embedded-postgres runs as a separate process altogether
+	// We must setup/teardown here, otherwise the process does
+	// not get properly cleaned up
+	done, err := testutils.TryGetPostgres()
+	defer done()
+	if err != nil {
+		fmt.Println(err.Error())
+		os.Exit(1)
+	}
+	exitVal := m.Run()
+	done()
+	os.Exit(exitVal)
+}
+
 func (s *Suite) SetupSuite() {
 
-	pg := embeddedpostgres.NewDatabase()
-	if err := pg.Start(); !assert.NoError(s.T(), err) {
+	sqlDb, err := gorm.Open(postgres.Open("host=localhost port=15432 user=postgres password=postgres dbname=postgres sslmode=disable"))
```

Comment:
can we set the port dynamically?

Reply:
Yeah we could add an env var like TEST_POSTGRES_PORT=... and read from it, with a default

Reply:
Yea